kernel/kernel-defaults/src/test/resources/workload_specs/basic_with_checkpoint_latest.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "read_metadata",
```

Comment:
can we also include a name? or do you want to just infer the name from the table path?

I do think a name string could be longer and more descriptive?

Reply:
Clarifying question: What's the differencce between a name and a description here?

Reply:
I'd consider a description to be something like what you proposed below: `Table schema=(ID LONG), protocol=(1,2), dataLayoutSpec=None, checkpoints=True`. Describes the workload in greater detail.

A name can give a high level description of the workload just like how a test name describes the high level of the test.